string name
float32 x
float32 theta
string base_color
string pendulum_color
---
bool result
